lib/BridgeList.php

```diff
@@ -96,6 +96,16 @@ private static function getSearchbar() {
 	private static function getFooter($totalBridges, $totalActiveBridges, $showInactive) {
 		$version = Configuration::getVersion();
 
+		$maintainer = Configuration::getConfig('misc', 'maintainer_email');
+		$maintainertext = '';
+		if (!empty($maintainer)) $maintainertext = <<<EOD
```

Comment:
The body of if-statements should be included with a block

```suggestion
		if (!empty($maintainer)) {
			$maintainertext = <<<EOD
```